internal/cli/server/config.go

```diff
@@ -1106,8 +1105,6 @@ func (c *Config) buildEth(stack *node.Node, accountManager *accounts.Manager) (*
 		log.Debug("Sanitizing Go's GC trigger", "percent", int(gogc))
 		godebug.SetGCPercent(int(gogc))
 
-		n.TrieCleanCacheJournal = c.Cache.Journal
-		n.TrieCleanCacheRejournal = c.Cache.Rejournal
```

Comment:
I can see that these 2 flags have been removed by geth (in `cmd/utils/flag.go`), I think we should remove these flags from bor as well.

Also, in `cmd/utils/flag.go` I can see that some new flags have also been introduced, should we add them to bor now, or we can do that later?